recruiter/views.py

```diff
@@ -73,3 +56,19 @@ def recruiterRegister(request):
 
 def recruiter_created_successfully(request):
     return render(request, 'recruiter/recruiter_created_successfully.html')
+
+
+def create_new_job_form(request):
+    recruiter_object = get_object_or_404(Recruiter, user_id=request.user.id)
```

Comment:
if it returns 404 your functions continues but then will fail/insert invalid data on lines 68-69

Reply:
If I understand correctly of what I read online, if the user isn't found then get_object_or_404 will raise a 404 error and the function will stop and won't continue

Reply:
Yes the 404 error is thrown as an exception so the function does not continue.